flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+    "only-arches": ["x86_64"],
+    "skip-icons-check": true
```

Comment:
Why though?

Reply:
only x86_64 because the build process requires an internet connection via CMake, so It's taking the binaries from a github actions

the icon check is being skipped because I was told to